lib/environment/inventory.js

```diff
@@ -17,6 +17,7 @@ const uiModules = [
   'ui-finance',
   'ui-orders',
   'ui-calendar',
+  'ui-tags',
```

Comment:
@mkuklis, embrace the CLI!

Reply:
oh nice! Thank you for catching this @ttamj!